vulnerability_report.md

```diff
@@ -0,0 +1,21 @@
+A1 injection:
```

Comment:
Your vulnerability report doesn't really follow any of the format outlined in the sample report. Please reformat and update.